src/App.tsx

```diff
@@ -10,33 +12,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+  const currentСomponent = tabs.find(({ id }) => id === selectedTabId);
```

Comment:
```suggestion
  const selectedTab = tabs.find(({ id }) => id === selectedTabId);
```
I think this name would be more accurate. 
Also, what if find methods finds nothing and return undefined? You should set the first tab in that case.
![image](https://user-images.githubusercontent.com/96409563/224535047-50426db6-f0f0-4399-9289-e13212c2cd2f.png)